docker-compose.yaml

```diff
@@ -28,6 +28,22 @@ services:
     init: true
     ipc: host
 
+  nginx:
+    image: nginx:1.18.0
+    env_file:
+      - env/.nginx.env
+    ports:
+      - "80:80"
+      - "443:443"
+    command: /bin/bash -c "envsubst '$$NGINX_HOST' < /etc/nginx/conf.d/conf.template > /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'"
+    volumes:
+      - ./nginx/server.conf:/etc/nginx/conf.d/conf.template
+      - ./nginx/nginx.conf:/etc/nginx/nginx.conf
+    depends_on:
+      - app
+    # #TODO Without this dependency the configuration of nginx can not use `app`
+    # as a hostname. Why?
```

Comment:
Not sure here, but if we proxy traffic the app we need the app container to be up.

And potentially we should wait-for-it on app:5000 (?)

Reply:
Yeah. I'm also unsure about this. But remove it, and suddenly nginx can not resolve the network `app` anymore

Reply:
So leave it in? :)